strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/layout/npm/NpmArtifactController.java

```diff
@@ -398,9 +413,15 @@ public ResponseEntity publish(@RepositoryMapping Repository repository,
                                   HttpServletRequest request)
             throws Exception
     {
+        if (nameContainsRevision(name))
+        {
+            return ResponseEntity.status(HttpStatus.OK)
+                                 .body("");
```

Comment:
Question: Do we actually need an empty `body` here?

Reply:
@carlspring Nope, I'll remove it

Reply:
Okay, cool, thanks! :)